mod/bbb-web/Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM mozilla/sbt:8u181_1.2.7 AS builder
 
-RUN apt-get update && apt-get install -y subversion
-
-# download bbb-common-message
-ENV TAG_COMMON_MESSAGE v2.3.0
-RUN svn checkout https://github.com/bigbluebutton/bigbluebutton/tags/$TAG_COMMON_MESSAGE/bbb-common-message /bbb-common-message \
-    && rm -rf /bbb-common-message/.svn
-
-# compile bbb-common-message
-RUN cd /bbb-common-message \
- && ./deploy.sh 
+ARG BBB_REPO_URL
+ENV BBB_REPO_URL $BBB_REPO_URL
+ARG BBB_BRANCH_OR_TAG
+ENV BBB_BRANCH_OR_TAG $BBB_BRANCH_OR_TAG
+
+# upgrade git so we have the sparse checkout and clone with filter feature
+RUN echo "deb http://ftp.us.debian.org/debian testing main contrib non-free" >> /etc/apt/sources.list \
+    && apt-get update \
+    && apt-get install -y git \
+    && apt-get clean all
+
+# clone only bbb-common-message and bbb-common-web and bigbluebutton-web subfolder
+RUN git clone --filter=blob:none --no-checkout $BBB_REPO_URL \
+    && cd bigbluebutton \
+    && git sparse-checkout set bbb-common-message bbb-common-web bigbluebutton-web \
```

Comment:
this command differs between
- `bbb-web`
- `fsesl-akka`
- `apps-akka`

leading to docker not being able to cache the built `bbb-common-message`

I always tried ensuring that all commands equal between these images as far down as passible, so that the build time and resulting disk usage stays as low as possible.

...but maybe there is also a better way...?^^